st.js

```diff
@@ -502,6 +502,13 @@ Mount.prototype._loadIndex = function (p, cb) {
       if (d.size === '-') name += '/'
       var showName = name.replace(/^(.{40}).{3,}$/, '$1..>')
       var linkName = encodeURIComponent(name)
+	  
+	  // encoded slashes are /
+	  linkName = linkName.replace(/%2f|%5c/ig, '/')
```

Comment:
keep indenting the same, have you used tabs here? should be 2-space indenting in line with the containing block